catalog/app/containers/Auth/messages.js

```diff
@@ -0,0 +1,281 @@
+import { defineMessages } from 'react-intl';
+
+const scope = 'app.containers.Auth';
+
+export default defineMessages({
+  notificationAuthLost: {
+    id: `${scope}.notificationAuthLost`,
+    defaultMessage: 'Authentication lost. Please sign in again.',
+  },
+  notificationAuthError: {
+    id: `${scope}.notificationAuthError`,
+    defaultMessage: 'Authentication error. Try again later.',
+  },
+
+  // Wrapper
+  wrapperFailureHeading: {
+    id: `${scope}.Wrapper.Failure.heading`,
+    defaultMessage: 'Authentication error',
+  },
+  wrapperFailureDescription: {
+    id: `${scope}.Wrapper.Failure.description`,
+    defaultMessage: 'Something went wrong. Try again.',
+  },
+  wrapperFailureRetry: {
+    id: `${scope}.Wrapper.Failure.retry`,
+    defaultMessage: 'Retry',
+  },
+
+  // SignUp
+  signUpHeading: {
+    id: `${scope}.SignUp.heading`,
+    defaultMessage: 'Sign Up',
+  },
+  signUpPassResetHint: {
+    id: `${scope}.SignUp.passResetHint`,
+    defaultMessage: "Dont't remember your password?",
+  },
+  signUpUsernameLabel: {
+    id: `${scope}.SignUp.usernameLabel`,
+    defaultMessage: 'Username',
+  },
+  signUpUsernameRequired: {
+    id: `${scope}.SignUp.usernameRequired`,
+    defaultMessage: 'Enter a username',
+  },
+  signUpUsernameTaken: {
+    id: `${scope}.SignUp.usernameTaken`,
+    defaultMessage: 'This username is already taken. {link}',
+  },
+  signUpUsernameInvalid: {
+    id: `${scope}.SignUp.usernameInvalid`,
+    // TODO: specify username requirements
+    defaultMessage: 'This username is invalid',
```

Comment:
'Username invalid'

Reply:
Yeah, ask calvin for requirements. I think it's 8 characters and that's it.

Reply:
@meffij could you please clarify?

Reply:
Currently, the only restriction is that the password must be at least 8 characters long.

Reply:
@meffij cool, but what are the requirements for username?